package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "learnyounode-golf",
+  "version": "1.0.0",
+  "description": "my attempt at solving [`learnyounode`](https://github.com/workshopper/learnyounode) with the least amount of code possible",
+  "scripts": {
+    "build": "node ./scripts/build.js",
+    "clean": "rm -r -f ./tmp/",
+    "install": "[ -e ./tmp/ ] || (git clone git@github.com:workshopper/learnyounode.git ./tmp && cd ./tmp && npm i)",
+    "test": "npm run install && npm run build && ./tmp/node_modules/.bin/workshopper-adventure-test ./tmp/"
```

Comment:
🤔 I don't think this actually tests the solutions themselves, this just runs the tests of the workshopper?